test/perf/src/test/test.js

```diff
@@ -293,16 +293,43 @@ describe('abacus-perf-test', () => {
       ]
     };
 
+    const token2 = {
```

Comment:
Can we rename the token to "systemReadToken" and the one above to something like "objectStorageWriteToken" (or shorter) to make the purpose of the tokens more obvious ?